src/sap.ui.core/test/sap/ui/core/demokit/sample/MessageManager/ODataBackendMessagesComp/webapp/controller/Employee.controller.js

```diff
@@ -0,0 +1,74 @@
+sap.ui.define([
+	"sap/ui/core/sample/MessageManager/ODataBackendMessagesComp/controller/BaseController",
+	"sap/ui/model/json/JSONModel",
+	"sap/m/MessageToast"
+], function (BaseController, JSONModel, MessageToast) {
+	"use strict";
+
+	return BaseController.extend("sap.ui.core.sample.MessageManager.ODataBackendMessagesComp.controller.Employee", {
+
+		onInit: function () {
+
+			//create a view model
+			var oViewModel = new JSONModel({
+				busy : false,
+				dirty : false
+			});
+			oViewModel.setDefaultBindingMode("TwoWay");
+			this.getView().setModel(oViewModel, "view");
+
+			this.getRouter().getRoute("employee").attachMatched(this._onRouteMatched, this);
+		},
+
+		_onRouteMatched : function (oEvent){
+			var oViewModel, oForm;
+
+			oViewModel = this.getView().getModel("view");
+
+			this.getView().bindElement({
+				path : "/Employees(3)",		//hard coded in this demo
+				events : {
+					change: function(){
+						var oBinding, oElementBinding, oModel;
+
+						oElementBinding	= this.getView().getElementBinding();
+						if (oElementBinding && !oElementBinding.getBoundContext()) {
+							MessageToast.show("No data - do something here...");
+						}else{
+							oModel = this.getView().getModel();
+							oBinding = new sap.ui.model.Binding(oModel, oElementBinding.getPath(), oElementBinding.getBoundContext());
```

Comment:
Please don't access the class via global namespace. Defining the 'Binding' class as dependency above would be cleaner.